packages/specification/schemas/requests/dispensing/Withdraw.yaml

```diff
@@ -0,0 +1,90 @@
+      - $ref: ../../components/Organization/DispenseNotificationOrganization.yaml
+  extension:
+    type: array
+    description: |
+      FHIR extension array to hold Repeat Information.
+      This identifies the instance of the eRD prescription that is being returned.
+      Given that all eRD prescriptions have the same prescription ID,
+      the combination of Prescription ID and repeat number identifies the issue to withdraw or return.
+    items:
+      $ref: ../../components/_extensions/EPSRepeatInformation.yaml
+  identifier:
+    description: The unique identifier for the Withdraw Request.
+    type: array
+    items:
+      $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/Rfc4122
+  status: 
+    type: string
+    example: in-progress
+  statusReason:
+    type: object
+    description: The reason the prescription is being withdrawn, which must be coded as per the Withdraw Reason vocabulary.
+    properties:
+      coding: 
+        type: array
+        items:
+          $ref: ../../components/_fragments/codes.yaml#/components/schemas/EpsTaskDispenseWithdrawStatusReason
+  intent:
+    type: string
+    description: The intent of the Medication Requests being withdrawn.
+    example: order
+  code: 
+    type: object
+    properties:
+      coding:
+        type: array
+        items:
+          $ref: ../../components/_fragments/codes.yaml#/components/schemas/TaskCode
+  groupIdentifier:
+    description: The Prescription ID of the prescription that is being withdrawn.
+    $ref: ../../components/_fragments/groupIdentifier.yaml
+  focus:
+    type: object
+    description: The release response which released the prescription that is currently being withdrawn.
+    properties:
+      identifier:
+        $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/Rfc4122
+  for:
+    type: object
+    properties: 
+      identifier:
+        $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/NhsNumber
+  authoredOn:
+    type: string
+    description: The time the prescription is being withdrawn to Spine.
+    format: date-time
+    example: "2020-12-21T17:03:20-00:00"
+  owner:
+    type: object
+    description: Who is responsible for actioning the Withdraw request.
+    properties:
+      identifier:
+        $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/OdsOrganizationCode
+  requester:
+    type: object
+    description: The dispenser withdrawing the prescription.
+    properties: 
+      reference:
+        type: string
+        example: "#16708936-6397-4e03-b84f-4aaa790633e0"
+  reasonCode:
+    type: object
+    description: The reason for the task, should only be included if there is no focus or differs from the reason indicated in the focus.
+    properties:
+      coding:
+        type: array
+        items:
+          $ref: ../../components/_fragments/codes.yaml#/components/schemas/ReasonCode
```

Comment:
We shouldn't use a standard import here, we should be indicating the valid list of reason codes in here.